src/chttpd/src/chttpd.erl

```diff
@@ -489,7 +489,7 @@ set_auth_handlers() ->
     AuthenticationDefault =  "{chttpd_auth, cookie_authentication_handler},
       {chttpd_auth, default_authentication_handler}",
     AuthenticationSrcs = couch_httpd:make_fun_spec_strs(
-        config:get("chttpd", "authentication_handlers", AuthenticationDefault)),
+        chttpd_util:get_chttpd_config(authentication_handlers, AuthenticationDefault)),
```

Comment:
This one will read from chttpd only as discussed in https://github.com/apache/couchdb/pull/3567/files#r635557609